packages/labs/context/src/lib/controllers/context-provider.ts

```diff
@@ -29,7 +29,28 @@ export class ContextProviderEvent<
   public constructor(public readonly context: C) {
     super('context-provider', {bubbles: true, composed: true});
   }
+
+  /**
+   * Whether the element that dispatched this event can take over subscriptions
+   * from an ancestor provider.
+   */
+  declare canTakeOverSubscriptions: boolean;
+
+  /**
+   * Ongoing subscriptions that one of our ancestors was handling that are now
+   * our responsibility.
+   */
+  movedSubscriptions: undefined | MovedSubscription<ContextType<C>>[] =
+    undefined;
 }
+// True for all ContextProviderEvents that we fire, but we need to set this
+// so that ancestor providers can tell whether the provider that dispatched this
+// event will actually take over any subscriptions that they move to
+// `event.movedSubscriptions`.
+// This way we fail gracefully in the case where there's version drift, or
+// another implementor that of the context protocol that also dispatches
```

Comment:
Nit: reword `another implementor that of the`. I think there is an erroneous `that`.

Reply:
Removed the code in question